apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -239,6 +239,7 @@ export class VersionAction extends BaseRushAction {
     });
 
     if (packageJsonUpdated) {
+      git.addChanges('.', this.rushConfiguration.versionPolicyConfigurationFile);
```

Comment:
Good catch. This issue was introduced with my PR #2272 for reference.

Reply:
glad to help 👍